src/Client.js

```diff
@@ -1,68 +1,18 @@
-import axios from 'axios';
-import url from 'url';
-
-function throwError(message) {
-  const error = new Error(message);
-  throw error;
-}
+import HTTP from './network/HTTP';
 
 export default class Client {
-  constructor({
```

Comment:
The current constructor is already suitable for our needs and it does goods verifications, why did you replace it?

Reply:
Oppsie, hehe... done :smile: